app/components/assign-point.js

```diff
@@ -3,68 +3,114 @@ import { computed } from '@ember/object';
 
 export default Component.extend({
   disabled: false,
-  canBeLess: true,
-  canBeMore: true,
-  onLess() {},
-  onMore() {},
+  min: 0,
+  max: 100,
+  value: 0,
+  step: 1,
+  onChange() {}, //Action to trigger when a new value has been selected
 
-  _canReallyBeLess: computed('canBeLess', 'disabled', function() {
-    return this.canBeLess && !this.disabled
+  _inputValue: '', //Only for display of yielded input field
+  init() {
+    this._super(...arguments)
+    this.set('_inputValue', this.value)
+  },
+
+  _canBeLess: computed('value', 'min', function() { return this.min < this.value }),
+  _canBeMore: computed('value', 'max', function() { return this.value < this.max }),
```

Comment:
You can drop the `canReallyBeLess` fields now and code it directly in `canBeLess`, as that's what you will use anyway.